site/docs/concepts/architecture.md

```diff
@@ -3,31 +3,26 @@
 Fybrik takes a modular approach to provide an open platform for controlling
 and securing the use of data across an organization. The figure below showcases the
 current architecture of the Fybrik platform, running on top of Kubernetes. 
-The storage systems shown in the lower half of the figure are merely an example.
 
 The core parts of Fybrik are based on Kubernetes controllers and Custom Resource Definitions (CRDs) in order to define its work items and reconcile the state of them.
 
-The primary interaction object for a data user is the `FybrikApplication` custom resource where a user defines which data should be used for which purpose. The following chart and description describe the architecture and components of Fybrik relative to when they are used.
+The primary interaction object for a data user is the `FybrikApplication` custom resource (1) where a user defines which data should be used for which purpose. The following chart and description describe the architecture and components of Fybrik relative to when they are used.
 
-![Architecture](../static/workflow_multicluster.svg)
+![Architecture](../static/architecture.png)
```

Comment:
Maybe remove the icon of the data governance officer, since none of the other actors are shown?

Reply:
done